core/src/main/java/edu/ucr/cs/riple/core/util/Utility.java

```diff
@@ -179,6 +180,11 @@ public static void writeNullAwayConfigInXMLFormat(FixSerializationConfig config,
       outputDir.setTextContent(config.outputDirectory);
       rootElement.appendChild(outputDir);
 
+      // UUID
```

Comment:
I wanted to do that but I think it might make it complicated, please consider the code below:
```java
Element suggestElement = doc.createElement("suggest");
suggestElement.setAttribute("active", String.valueOf(config.suggestEnabled));
suggestElement.setAttribute("enclosing", String.valueOf(config.suggestEnclosing));
rootElement.appendChild(suggestElement);
```

If we follow the extract method, we should have a method with signature `addElementWithAttribute(Document, ElementName, TagName, TagValue)` which with the above code is not possible to use this method since it is adding multiple tags to the same element so it either has to look for an existing element with each call (and only add it to `rootElement` as a child if its not added before) or pass a `Map` of tag name/value to the `addElementWithAttribute` method to support the code above.  Which with both version, I think reading the code at the current state will give an idea how the `xml` will look like clearer.